app/lib/frontend/handlers/routes.dart

```diff
@@ -283,3 +266,37 @@ class PubSiteService {
   @Route.get('/packages.json')
   Future<Response> packagesJson(Request request) => packagesHandler(request);
 }
+
+/// Routes that are only processed by the old pub domain.
```

Comment:
Can we put this in a separate file...

Reply:
I'd like to keep it in the same file, I think it is really nice to have a single file for all the routes we have.